lookout/style/format/tests/test_analyzer_integration.py

```diff
@@ -113,14 +113,14 @@ def setUpClass(cls):
         parent = Path(__file__).parent.resolve()
         cls.base_dir_ = tempfile.TemporaryDirectory()
         cls.base_dir = cls.base_dir_.name
-        cls.jquery_dir = os.path.join(cls.base_dir, "jquery")
-        # str() is needed for Python 3.5
         with tarfile.open(str(parent / "jquery.tar.xz")) as tar:
-            tar.extractall(path=cls.base_dir)
+            tar.extractall()
+        # str() is needed for Python 3.5
+        cls.jquery_dir = str(Path(parent) / "jquery")
```

Comment:
yes ofc, but removing it with `shutil.rmtree` is also fine, no ?
Because I need to access those files later in the tests, and with a `TemporaryDirectory`, they disappeared once we get out of the `setUpClass` method.

Reply:
well, by `Ctrl+C` I mean control-break signal, so, you did not run code in `tearDownClass'.
And if you use the temporary directory the same happens but I believe it will be clean up by the system later.

What about the issue you mentioned try to use
```python
cls.base_dir_ = tempfile.mkdtemp()
```
> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.

https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

Reply:
ok got it, let me try that!